README.md

```diff
@@ -180,8 +180,7 @@ ils](#details-cabal) |
 * <a name="details-haskell-workshop"></a>
   `haskell-workshop/`
 
-  Archival page.  Not updated since 2008.  Source is linked as
-  http://abridgegame.org/darcs/ but that link seems long-dead.
+  Archival page.  Not updated since 2008.
```

Comment:
Can we undo this change please? This table tracks contact information for upstream sources, and it's important to know when the upstreams are dead because that gives us more justification for potentially incorporating the subsites directly into this repo.

Reply:
Since this branch is editable by maintainers, I went ahead and did that.

Reply:
My main objective with this change was to remove that hyperlink which currently redirects to a sports betting site.  If you want to keep the content, perhaps it could be demoted from a hyperlink to plain text?  Because I don't think haskell.org should link to sports betting sites...

Reply:
Aha, I see. I agree. Does anyone know offhand of the way to avoid link-like text turning into hyperlink in Github-flavoured Markdown?

Reply:
Perhaps wrap it in backticks (to make it monospace)?